templates/records/record_detail.html

```diff
@@ -10,48 +10,39 @@
 
 {% block content %}
 
-    {% if page.parent %}
-        {% include "includes/records/hierarchy-local.html" %}
-    {% endif %}
+    {% include "includes/records/record-cta-panel.html" %}
 
-    <div class="row no-gutters">
+    <div class="container">
 
-        <div class="col-xl-3 order-xl-2 specific-border-bottom-lg specific-sticky specific-bg-grey-5">
-              {% include "includes/records/hierarchy-global.html" %}
-        </div>
+        {% include "includes/records/record-title.html" %}
+        {% include "includes/records/hierarchy-global.html" %}
 
-        <div class="col-xl-9 order-xl-1 specific-padding-bottom-lg">
-            {% include "includes/records/record-title.html" %}
-            <div class="container">
-                {% include "includes/records/key-facts.html" %}
+        <h2 id="record-details-heading">Description and record details</h2>
```

Comment:
> We will need to do full housekeeping of details page associated files after the details page has deployed. This is to avoid any further delay in the details page being available to the team.

`{% include "includes/records/key-facts.html" %}` is removed

Could we remove ones that are removed by this branch, and after that any other from a separate ticket for this activity?

Reply:
[DF-464](https://national-archives.atlassian.net/browse/DF-464) Housekeeping for details page is created for this.